### ISSUE TITLE
SSL Error using Swift 3 branch

### ISSUE THREAD

**Issue by mercury** [nuclearace/Socket.IO-Client-Swift#129]
I am getting the error `ERROR SocketEngine: An SSL error has occurred and a secure connection to the server cannot be made.` when tried with new Xcode 8 and Swift 3. I was trying to connect to non SSL URL but that shouldn't be a problem as I was able to make REST calls to the same URL.

**Comment by nuclearace** [comment]
I got that too on OS X, I think it's Apple's security enforcing SSL. Not sure how to turn it off.

**Comment by mercury** [comment]
Right. even adding exceptions to Info.plist doesnt help. Not sure why normal Http requests would work but not this. Please post if you find any workaround..
